### ISSUE TITLE
RSS ordering causing issues

### ISSUE THREAD

**Issue by alexandru** [steveklabnik/steveklabnik.com#11]
I'm using Newsblur as my feed reader, and the [RSS feed](https://steveklabnik.com/feed.xml) is causing issues because items aren't ordered in descending order of `pubDate`. Changing the feed to be ordered might make life easier for clients that depend on ordering.

https://forum.newsblur.com/t/issues-with-feed-steveklabnik-com/10372

Thanks for your writing, and your work in general.

**Comment by steveklabnik** [comment]
Interesting! That is... unfortunate, but also, fixing it isn't a major problem. I don't know when I will get around to this but I think it should be fixed.

**Comment by steveklabnik** [comment]
I went to fix this this morning, because I have a new post, but it's not super simple. Astro has updated since I wrote this code, and 

     items: import.meta.glob('./**/*.md'),

isn't an array of objects, so it's not as simple as calling `sort` there. 

I don't expect this to be super difficult, but I am out of time right now to work on it, and so mostly wanted to say "hey sorry I am making a new post without fixing this," haha. Maybe I can later today...
